src/Common/src/Interop/Windows/Winsock/Interop.WinsockAsync.cs

```diff
@@ -34,9 +34,12 @@ internal struct ControlDataIPv6
         [StructLayout(LayoutKind.Sequential, CharSet = CharSet.Unicode)]
         internal struct WSAPROTOCOLCHAIN
         {
-            internal int ChainLen;                                 /* the length of the chain,     */
+            // The length of the chain.
+            internal int ChainLen;
+
+            // A list of catalog entry IDs.
             [MarshalAs(UnmanagedType.ByValArray, SizeConst = 7)]
-            internal uint[] ChainEntries;       /* a list of dwCatalogEntryIds */
```

Comment:
@stephentoub, @pgavlin  Are we considering ever changing these to the actual SDK names? If yes, I would prefer if we leave comments like the one above in - it makes search through the SDK much simpler.